android/src/main/AndroidManifest.xml

```diff
@@ -1,6 +1,3 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Why was the package name changed? And does this have downstream impact?

Reply:
Did it change?

Reply:
Sorry, I quoted the wrong line, but yep, the line below:
`com.getcapacitor.community.intercom.intercom`
changed to
`com.joinflux.flux.intercom`

Reply:
>My main rationale was to ensure consistency across our plugins, and as for whether it has downstream impact I don’t think so. The versioning has changed to indicate that there are breaking changes and one of those would be the fact that the package name needs to be updated within `capacitor.plugins.json`, which happens automatically when updating the plugin and syncing. It’s mainly used to locate where the class is for the plugin on Android